assets/base/i18n/en.json

```diff
@@ -1,121 +1,4 @@
 {
-  "about.date": "Build Date:",
-  "about.enterpriseEditione1": "Enterprise Edition",
```

Comment:
Why have those keys been deleted ? Are we going to re-add them as we bring in more components ?

Reply:
This is done automatically by the extract tool.. so as we move forward they'll get re-added